api/src/main/resources/workbench-api.yaml

```diff
@@ -235,6 +235,35 @@ paths:
       responses:
         204:
           description: Request Received.
+
+  "/v1/getTerraTermsOfServiceStatus":
+    get:
+      tags:
+        - profile
+      description: Retrieves if the logged in user has accepted Terra service or not
+      operationId: getUserTerraTermsOfServiceStatus
+      responses:
+        200:
+          description: Returns false if user is required to accept Terra Terms of Service, else true
+          schema:
+            type: boolean
+
+  "/v1/acceptTermsOfService":
+    post:
+      tags:
+        - profile
+      description: User accepts AOU and Terra Terms of Service
```

Comment:
need to clarify that this is *a specific version* of the AoU TOS but *the current version* of the Terra TOS

Reply:
Updated description